pandas/tests/window/test_window.py

```diff
@@ -29,14 +29,19 @@ def test_constructor(self, which):
         c(win_type="boxcar", window=2, min_periods=1, center=False)
 
         # not valid
+        msg = (
+            "min_periods must be an integer|"
+            "center must be a boolean|"
+            "Invalid win_type"
+        )
```

Comment:
can we narrow this down for any of the three cases below?

we're moving towards a pattern where this would be 

```
msg = "|".join([
    "min_periods must be an integer",
     ...
])
```

Reply:
Rather than putting these all into one pattern I think can split up the loops. `min_periods must be an integer` would really only apply to the passing of "foobar" to `win_type` so just break that loop and pass the appropriate message

Reply:
I have adjusted it as per the style guide.